pkg/tests/end_to_end_tests/vacuum_test.go

```diff
@@ -12,6 +12,21 @@ import (
 	"github.com/timescale/promscale/pkg/vacuum"
 )
 
+const (
+	sqlChunksToFreeze = `
```

Comment:
Nit: these two queries different only in selector so maybe we can template and format it by pasing in what to select

Reply:
https://github.com/timescale/promscale/pull/1761/commits/7d1b0a97e9d0135b227d0a20b7976810e519bee9